### ISSUE TITLE
Forgo the reading of the fishcam by the server. 

### ISSUE THREAD

**Issue by Wilkuu-2** [saproto/saproto#1887]
The current fishcam implementation opens the file on the server once per user 
and probably causes strain on the server. 

The simplest solution is to let the frontend access the webcam instead.
But sadly, CORS exists and does not want our server to live a happy life.
The simple solution only works on Firefox.

The CORS problem seems to have a root on the webcam,
namely, the webcam requires authentication for OPTIONS request, [which is not included in preflight requests.](https://stackoverflow.com/questions/59824175/do-cors-preflight-requests-include-the-requests-headers)

Firefox seems to dodge having to preflight at all for some reason.
Chrome does not.

TL;DR: The webcam has a skill issue regarding CORS.

A solution to this could be passing the webcam through a proxy of some sorts on the side of the server, bypassing CORS completely.
That means that the server is still involved, but doesn't have to read the request.

Another way would be reading the fishcam outside the API request and returning the same copy to multiple users, while making the implementation a bit less hacky, hopefully.

**Comment by Wilkuu-2** [comment]
Regarding the Proxy solution,
Laravel 9 has the TrustedProxy package integrated.
That means that proxying the webcam can be done without dependencies after #1808 is merged.

**Comment by Wilkuu-2** [comment]
> Regarding the Proxy solution, Laravel 9 has the TrustedProxy package integrated. That means that proxying the webcam can be done without dependencies after #1808 is merged.

TrustedProxy is not for making proxies, but for laravel being accessed by proxy, that is a mistake on my side. 

I also narrowed down the problem with the current implementation: 
* The implementation using fread does not impact memory negatively.
* The bandwith is strongly affected
* Every time a client opens the fishcam, the server opens the file once more. 

If we want to proxy this, we need to do that through Apache rather than laravel/php
